src/postListUtil.js

```diff
@@ -0,0 +1,48 @@
+import { request } from "../Api/api.js";
+
+export const createPost = async (parent = null) => {
+  return await request("/documents", {
+    method: "POST",
+    body: JSON.stringify({
+      title: "",
+      parent: parent,
+    }),
+  });
+};
+
+export const removePost = async (id) => {
+  await request(`/documents/${id}`, {
+    method: "DELETE",
+  });
+};
+export const getPost = async (id) => {
+  return await request(`/documents/${id}`, {
+    method: "GET",
+  });
+};
+
+export const renderLogic = (arr) => {
+  return `
+  <ul>
+    ${arr
+      .map(({ id, title, documents }) => {
+        return documents
+          ? `<li data-id="${id}">
+            <div class="document" data-id="${id}" data-name="list">
+              <span data-name="span-list" data-id="${id}">> ${title}</span>
+              <button data-id="${id}" data-name="plus"> + </button> 
+              <button data-id="${id}" data-name="minus"> x </button>
+            </div>
+          </li>` + renderLogic(documents)
```

Comment:
저도 다연님 의견 동의합니다! 아니면 해당 dom을 template으로 따로 빼거나, 맨 마지막 줄에만 삼항 연산자를 써도 될 것 같아요 
아니면 HTML을 통째로 바꾸는 것이 아닌  insertAdjacentHTML로 해당 요소 뒤에 추가하는 방식도 있어요 😊
```suggestion
    ${arr
      .map(({ id, title, documents }) => 
      `<li data-id="${id}">
            <div class="document" data-id="${id}" data-name="list">
              <span data-name="span-list" data-id="${id}">> ${title}</span>
              <button data-id="${id}" data-name="plus"> + </button> 
              <button data-id="${id}" data-name="minus"> x </button>
            </div>
          </li>` + documents ? renderLogin(documents) : ''
```